poll/public/js/poll_edit.js

```diff
@@ -25,7 +25,9 @@ function PollEditUtil(runtime, element, pollType) {
                             // The degree of precision on date should be precise enough to avoid
                             // collisions in the real world.
                             var bottom = $(button_mapping[context_key]['bottomMarker']);
-                            var new_item = $(self.answerTemplate(button_mapping[context_key]['itemList']));
+                            var new_item_dict = self.extend({}, button_mapping[context_key]['template']);
+                            new_item_dict['key'] = Date.now();
+                            var new_item = $(self.answerTemplate({'items': [new_item_dict]}));
```

Comment:
Can you explain the fix? It's not clear to me from the code what the issue was, and how this is addressing it.